prime-router/src/main/kotlin/fhirengine/translation/hl7/ConfigSchemaProcessor.kt

```diff
@@ -43,6 +44,7 @@ abstract class ConfigSchemaProcessor<
      */
     abstract fun process(
         input: Original,
+        reportEventService: IReportStreamEventService? = null,
```

Comment:
This is the main issue I have with this (aside from it not working at the moment). Opening up these classes to add in the event service feels too messy.